src/server/redis_connection.h

```diff
@@ -180,6 +172,15 @@ class Connection : public EvbufCallbackBase<Connection> {
   std::set<std::string> watched_keys;
   std::atomic<bool> watched_keys_modified = false;
 
+  bool CheckClientReachOBufLimits(size_t reply_bytes);
```

Comment:
In all functions of this "family" we can expand  a single `O` letter to `Output`